themes/CmsForNerd/pager.php

```
@@ -4,25 +4,38 @@
+ *
+ * The `view` query parameter selects AMP rendering; all other values use the standard layout.
+ *
+ * @param CmsForNerd\CmsContext $ctx The immutable rendering context.
  */
 function pager(CmsForNerd\CmsContext $ctx): void
 {
     $viewMode = $_GET['view'] ?? 'standard';
 
+    // [PERFORMANCE] Start static page cache interception
+    if (class_exists('\\CmsForNerd\\PerformanceUtils')) {
+        \CmsForNerd\PerformanceUtils::startPageCache($ctx->scriptName, $viewMode);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\CmsForNerd\PerformanceUtils' in method 'pager'. (undefined)

(StaticAccess)